Dockerfile.kube-client-agent.openshift

```diff
@@ -0,0 +1,17 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
+
+COPY . /go/src/github.com/coreos/kubecsr
+
+RUN cd /go/src/github.com/coreos/kubecsr \
```

Comment:
optional: you can simplify it by setting the workdir
```Dockerfile
WORKDIR /go/src/github.com/openshift/kubecsr
COPY . .
RUN make bin/kube-client-agent
```

Reply:
good advice thanks!